specfile/specfile.py

```diff
@@ -179,11 +269,26 @@ def rpm_spec(self) -> rpm.spec:
 
     def reload(self) -> None:
         """Reloads the spec file content."""
-        self._lines, self._trailing_newline = self._read_lines(self.path)
+        try:
+            path = Path(self._file.name)
```

Comment:
Here we are treating `self._file` as named file again.

```suggestion
            path = Path(cast(FileIO, self._file).name)
```